plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -957,12 +957,7 @@ protected function column_comment( $item ) : void {
 			echo $in_reply_to . '<br><br>'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		}
 
-		printf(
-			'%1$s%2$s%3$s',
-			'<div class="comment-text">',
```

Comment:
`comment_text()` wrapper the comment in a paragraph tag without any class. We're using a dive with the `comment-text` class. Should we keep the wrapper element?

Reply:
> It is worth pointing out that this PR impacts the DOM structure of the page by removing the selector .comment-text. I checked on[ wpdirectory](https://wpdirectory.net/search/01JDW6C6CB3TPZKSPGMT8TSFA1), and it looks like that no extension rely on this selector on the editor side.

Yeah, I mentioned this in the PR description. Core doesn't wrap the comment, but if you think that we should do, I'm happy to add this. No strong preference.

Reply:
I think I'd put it back just in case. Not much work to bring it back and nearly no gain by removing it. 🙌

Reply:
Oh, interesting! We have some unit tests that are failing:

![image](https://github.com/user-attachments/assets/291135f3-a44e-4e25-95f5-1f0354c3f2f0)